1.11/release-notes/1.11.0-beta3/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.0 Beta 3
+title: Release Notes for 1.11.0 Beta 3
+menuWeight: 0
+excerpt:
+---
+
+<table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;">
+
+<h3>This beta release is for testing only and not to be used in production. It will only support new installations.</h3>
+<h3>DC/OS 1.11.0 Beta 3 has a number of limitations that will be resolved at GA time:</h3>
+
+<ul>
+<li>Upgrades from 1.10 are not supported.</li>
+<li>DC/OS 1.11 requires CLI version 0.6.x.
+  <ul>
+  <li><a href="/1.11/cli/uninstall/">Uninstall the existing CLI</a>.</li>
+  <li>Install version 0.6.x using the <strong>Install CLI</strong> instructions in the dropdown in the upper left hand corner of the 1.11 DC/OS GUI.</li>
+  </ul>
+<strong>Note:</strong> CLI version 0.6.x is not compatible with DC/OS 1.10</li>
+</ul>
+
+<a name="new-features"></a>
+# New features and capabilities
+
+## Apache Mesos 1.5 and Marathon 1.6 Integrated.
+- DC/OS 1.11 is is based on Mesos 1.5. View the [changelog](https://github.com/apache/mesos/blob/master/CHANGELOG).
+
+- DC/OS 1.11 is integrated with the latest release of Marathon, version 1.6. Resulting breaking changes and new features are documented below. For more information about Marathon 1.6, consult the [Marathon changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
+
+## Networking
+- IPv6 is now supported for Docker containers.
+  You can now launch docker containers through Marathon on an IPv6 Docker user network. Tthe built-in DC/OS overlay also supports IPv6 virtual networks for Docker containers. By default, DC/OS overlay now comes with an IPv6 virtual network called `dcos6`. You can use `dcos6` to launch IPv6 docker containers through Marathon.
+  **Note:** IPv6 is not supported for CNI, and hence UCR containers.
+- Edge-LB 1.0 RC candidate. [enterprise type="inline" size="small" /]
+  - Strict security mode is now supported.
+- Performance improvements to the DC/OS network stack.
+  All networking components (minuteman, navstar, spartan) are aggregated into a single Erlang VM called `dcos-net`. This improves the resource consumption of the Erlang components and greatly improves reliability and availability to these components.
```

Comment:
Nit: inconsistent usage of backticks. `dcos-net` has backticks, but minteman, navstar, and spartan don't?